### PR TITLE
Bump supercluster version

### LINKED ISSUE #524: Cluster layer goes blank
**Describe the bug**
As said the `Cluster Layer` goes blank when zoom out at small zoom level, the official demo can reproduce. 

Change the `earthquakes` demo point layer to cluster
image

 Then zoom out.
image

### PR DESCRIPTION
**Pull request by akre54**
Fixes #524 

As a bonus we're now on the same version as mapbox-gl!

### CODE REVIEW

File: src/deckgl-layers/layer-utils/cluster-utils.js [hunk 1]
```diff
@@ -41,11 +41,9 @@ export function getGeoJSON(data, getPosition) {
 const clusterResolver = ({clusterRadius}) => `${clusterRadius}`;
 
 const getClusterer = memoize(({clusterRadius, geoJSON}) => {
-  return supercluster({
+  return new Supercluster({
     maxZoom: 20,
     radius: clusterRadius,
-    initial: () => ({points: []}),
```
Comment (heshan0131) [inline src/deckgl-layers/layer-utils/cluster-utils.js]: this is a initial state for hovered poitns, and is used to calculate hover state. Is there a new prop that replacing it?

File: src/deckgl-layers/layer-utils/cluster-utils.js [hunk 2]
```diff
@@ -41,11 +41,9 @@ export function getGeoJSON(data, getPosition) {
...
-    map: props => props.data,
```
Comment (heshan0131) [inline src/deckgl-layers/layer-utils/cluster-utils.js]: why is this deleted without a replacement?

File: src/deckgl-layers/layer-utils/cluster-utils.js (hunk 1 above)
Comment (akre54) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: v6.0.0 [removed the initial](https://github.com/mapbox/supercluster/releases/tag/v6.0.0) argument. It now uses the first item as a basis. The hovered points should still work, no?

File: src/deckgl-layers/layer-utils/cluster-utils.js (hunk 2 above)
Comment (akre54) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: breaking change to the library: https://github.com/mapbox/supercluster/releases/tag/v6.0.0

File: src/deckgl-layers/layer-utils/cluster-utils.js (hunk 1 above)
Comment (heshan0131) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: According to docs:
```
In most cases, you can simply remove the initial option and things will work. There are some exceptions where you'd need to update your map function to make up for lack of initial, but they should be rare.
```
We should then update the map function, see comment below

Comment (heshan0131) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: originally initial function add an empty array of points. then the reduce function will push props.data to the `points` array. 

We need the property `points` in each cluster to store all the points that it contains. It is used to render hover info. Please make sure they are propogated

File: src/deckgl-layers/layer-utils/cluster-utils.js [hunk 3]
```diff
@@ -41,11 +41,9 @@ export function getGeoJSON(data, getPosition) {
...
     reduce: (accumulated, props) => {
```
Comment (heshan0131) [inline src/deckgl-layers/layer-utils/cluster-utils.js]: you can probably add the initial points here
```
if (!Array.isArray(typeof accumulated.points)) {
  accumulated.points = [];
}
```

File: src/deckgl-layers/layer-utils/cluster-utils.js (hunk 1 above)
Comment (akre54) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: It pulls the `points` property from the geoJSON object: https://github.com/keplergl/kepler.gl/blob/abe8e9b8e20d4f3b430c2c3c7d7350c838d49907/src/deckgl-layers/layer-utils/cluster-utils.js#L30

As far as I can tell it works correctly? I can see it changing the color of the hovered icon:

kepler-cluster-hover

If I'm missing something, how can I reproduce?

Comment (heshan0131) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: it should show total points correctly
Screen Shot 2019-06-24 at 2 05 56 PM

Comment (heshan0131) [reply src/deckgl-layers/layer-utils/cluster-utils.js]: You should keep the `map: props => props.data` function
